docs/admin-api-tools.md

```diff
@@ -17,7 +17,7 @@ You can manage Pulsar entities through the Pulsar admin layer via one of the fol
 
    - [REST API](https://pulsar.apache.org/admin-rest-api/?version=master): HTTP calls, which are made against the admin APIs provided by brokers. In addition, both the Java admin API and pulsar-admin CLI use the REST API.
 
-- [pulsar-admin CLI](https://pulsar.apache.org/reference/#/next/pulsar-admin/): It’s a command-line tool and is available in the bin folder of your Pulsar installation.
+- [pulsar-admin CLI](https://pulsar.apache.org/reference/): It’s a command-line tool and is available in the bin folder of your Pulsar installation.
```

Comment:
@urfreespace AFAIR, when the docs were in the `pulsar` repo, we can omit the `https://pulsar.apache.org` if it's an internal link. Just wondering if it is still true now since I cannot preview to verify this in my local build.

Reply:
Confirmed with @urfreespace - both are correct. So keep it as it is at this moment.